### DISCUSSION TITLE
How to set random seed in Sionna?

### DISCUSSION THREAD

**Discussion by communicationDL1** [NVlabs/sionna#304] [Q&A]
When I run this example（https://nvlabs.github.io/sionna/examples/MIMO_OFDM_Transmissions_over_CDL.html#）, I notice that the results vary with each execution. How should I set a random seed in Sionna for this case?

**Comment by jhoydis** [comment]
Hi,

You can simply set ``tf.random.set_seed`` before executing a specific cell. However, note that this does not work when XLA is activated (see this [link](https://www.tensorflow.org/xla/known_issues)).

Also note that setting a specific seed is not needed for Monte Carlo simulations with sufficient iterations (such as BER curves) since the results should be essentially independent of the seed.
